schema_based_on_diagram.sql

```diff
@@ -0,0 +1,56 @@
+CREATE DATABASE vetClinic;
+
+CREATE TABLE owners(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  full_name VARCHAR(20),
+  age INT,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE species(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE animals(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  date_of_birth DATE,
+  escape_attempts INT,
+  neutered BOOLEAN,
+  weight_kg DECIMAL,
+  species_id INT REFERENCES species(id),
+  owner_id INT REFERENCES owners(id),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE vets(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE specializations(
```

Comment:
- You have done great so far but there are still some project requirements you need to meet :+1: 

- DATABASE-DIAGRAM:
![Screenshot from 2023-09-05 22-50-14](https://github.com/MasumaJaffery/Vet-Clinic/assets/107632780/ec00dfa5-11a2-495d-ae30-bfa555e873d4)


- [ ] Just as the previous reviewer mentioned, you are meant to follow the diagram above and create all the tables and columns in it. You are meant to create the `medical_histories, treatments, invoice, invoice_items, patients` table and all the columns in them. and also implement all the relationships both by using foreign key and join tables. I'lll suggest you go through the project requirement carefully and implement this. :smile: :wave:

Reply:
![image](https://user-images.githubusercontent.com/107632780/265834480-ec00dfa5-11a2-495d-ae30-bfa555e873d4.png)
- [ ] Just like the previous reviewer has pointed out, please follow the attached diagram above ☝️ and create all the tables and their respective columns. Please see the project requirements below 👇 
![image](https://github.com/MasumaJaffery/Vet-Clinic/assets/46839548/03fd9c94-e18b-4f6e-aade-38c2fdeca714)